test/python/common.py

```diff
@@ -50,7 +50,7 @@ def with_ngraph(self, l, config=tf.ConfigProto()):
 
         os.environ['NGRAPH_TF_DISABLE_DEASSIGN_CLUSTERS'] = '1'
         ngraph_bridge.enable()
-        with tf.Session(config=config) as sess:
+        with tf.Session(graph=graph, config=config) as sess:
```

Comment:
Why do we need this change? Just Curious.

Reply:
when we write python code like `a = tf.constant(...); b = tf.constant(...); c = a+b`, this underlying graph is added to the default graph. When we read it from a pbtxt (like in this case), the graph is not added to default graph. So was passing the graph along to `common.py` to use during session construction.

But found a way to set the graph read from pbtxt as default graph (`as_default`)